packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -72,7 +72,7 @@
 							:is="registry.getNode(node.operationType)"
 							:node="node"
 							@append-output="(event: any) => appendOutput(node, event)"
-							@append-input-port="(event: any) => appendInputPort(node, event)"
+							@append-input-port="(event: any) => onAppendInputPort(node, event)"
```

Comment:
you can call `appendInputPort` from the service here directly

Reply:
yes!